packages/web-app-files/src/components/Collaborators/Collaborator.vue

```diff
@@ -121,10 +121,10 @@
             </div>
             <div v-if="isIndirectShare">
               <oc-tag
+                v-oc-tooltip="viaTooltip"
```

Comment:
Needs an `aria-label` for the `viaTooltip`

Reply:
Should be there now